code/mob/living/carbon/human/procs/get_desc.dm

```diff
@@ -346,7 +346,11 @@
 				. += "<br><span class='alert'>[src.name] has a blank expression on [his_or_her(src)] face.</span>"
 
 			if (!src.client && !src.ai_active)
-				. += "<br>[src.name] seems to be staring blankly into space."
+				if (find_player(src.last_ckey)?.client) // using vr goggles or something similar
```

Comment:
Won't this catch stuff like being respawned or mindswapped into a different body as well? Probably best to specifically check if they are in VR or using a scuttlebot (or have those things set a flag on the old body so we can tell)

Reply:
Yeah I think so, fixed

Reply:
Though it won't work for if you mindswap into a scuttlebot while controlling it. That's a pretty rare case though, maybe could be considered a "stuck in vr" type feature?